image-mirror.sh

```diff
@@ -142,10 +142,10 @@ function mirror_image {
       ARCH=$(jq -r '.architecture' <<< ${CONFIG})
       DIGEST=$(jq -r '.config.digest' <<< ${MANIFEST})
       if grep -wqF ${ARCH} <<< ${ARCH_LIST}; then
-        if copy_if_changed "${SOURCE}:${TAG}" "${DEST}:${TAG}-${ARCH}" "${ARCH}"; then
-          echo -e "\tAdding ${DEST}:${TAG}-${ARCH} => ${DEST}:${TAG}"
-          docker buildx imagetools create --tag ${DEST}:${TAG} ${DEST}:${TAG}-${ARCH}
-        fi
+        copy_if_changed "${SOURCE}:${TAG}" "${DEST}:${TAG}-${ARCH}" "${ARCH}"
+        echo -e "\tAdding ${DEST}:${TAG}-${ARCH} => ${DEST}:${TAG}"
```

Comment:
We now do this only if digests changed, so do we need these `echo Adding` lines? copy_if_changed logs all necessary info for both changed and unchanged.

Reply:
Glad to get rid of it if nobody will miss it!